tests/e2e/vsan_test.go

```diff
@@ -36,6 +38,7 @@ import (
 )
 
 const (
+	vsanTest       = "vsan_test"
 	vsanVolumeTest = "vsan_vol"
 	vsanPolicyFlag = "vsan-policy-name"
```

Comment:
I meant for `vsan-policy-name`, shouldn't it be moved to `tests/constants/admincli/vsanpolicy.go`?